tests/hotplug/cpu.go

```diff
@@ -235,6 +235,10 @@ var _ = Describe("[sig-compute][Serial]CPU Hotplug", decorators.SigCompute, deco
 
 			tests.ExpectMigrationSuccess(virtClient, migration, tests.MigrationWaitTime)
 
+			By("delete the migration object from the namespace")
```

Comment:
Wouldn't it be better to fix the loop below which should pick the correct migration?

Reply:
@xpivarc the problem is that we don't annotate/label migrations that were caused by hotplug. I can label the manual migration and filter it.

Reply:
Above you assert for the first migration to succeed. So there should be one migration that is not finalized.

Reply:
@xpivarc but it could be flaky, it can finish before I fire the API call

Reply:
If both are going to be finished that's okay as well.
`tests.ExpectMigrationSuccess(virtClient, migration, tests.MigrationWaitTime)` will succeed and everything else should be fine. Am I missing anything?